__tests__/unit/lib/utils/repoGitDiff.test.js

```diff
@@ -12,25 +12,26 @@ describe(`test if repoGitDiff`, () => {
       stdout: '',
     }))
     // eslint-disable-next-line no-undef
-    const work = repoGitDiff({ output: '', repo: '' }, globalMetadata)
+    const work = repoGitDiff({ output: '', repo: '', ignore: '__tests__/unit/lib/utils/.forceignore' }, globalMetadata)
     expect(work).toStrictEqual(output)
   })
 
   test('can resolve deletion', () => {
+
     const output = [
-      'D      force-app/main/default/objects/Account/fields/awesome.field-meta.xml',
+      'D      force-app/main/default/objects/Account/fields/awesome.field-meta.xml'
     ]
     child_process.spawnSync.mockImplementation(() => ({
       stdout: output[0],
     }))
     // eslint-disable-next-line no-undef
-    const work = repoGitDiff({ output: '', repo: '' }, globalMetadata)
+    const work = repoGitDiff({ output: '', repo: '', ignore: '__tests__/unit/lib/utils/.forceignore' }, globalMetadata)
```

Comment:
use a constant at the beginning of the file for the path to the forceignore file plz